model/checkinconfig.go

```diff
@@ -0,0 +1,18 @@
+package model
+
+// CategoryCodes is an array of category codes
+type CategoryCodes struct {
+	CategoryCodes []string `json:"category_codes"` // CategoryCodes is an array of category codes
+}
```

Comment:
Maybe I'm wrong, but I thought we already had a structure for doing this? If we do not, then I suggest it may be a good idea to call this structure `CheckInCategoryCodes`.

Reply:
There is no other CategoryCodes struct

Reply:
So, I suggest either you put it into `model/categorycodes.go` or prefix `CheckIn` to it?

Reply:
Now that I take another look at the code, I believe we should just have `model/checkin.go` with everything inside it using the `CheckIn` prefix to make it super easy to understand where the code lives!